05_parameters.md

```diff
@@ -284,166 +282,43 @@ also increase the chance of running the right technique. Predicting which
-    update_gap_integral]
-- `solver.parameters.num_workers = 32`:
-  - 15 full problem subsolvers: [default_lp, lb_tree_search, max_lp, no_lp,
-    objective_lb_search, objective_lb_search_max_lp, objective_lb_search_no_lp,
-    objective_shaving_search_max_lp, objective_shaving_search_no_lp, probing,
-    probing_max_lp, pseudo_costs, quick_restart, quick_restart_no_lp,
-    reduced_costs]
-  - 15 first solution subsolvers: [fj_long_default, fj_long_lin_default,
-    fj_long_lin_random, fj_long_random, fj_short_default, fj_short_lin_default,
-    fj_short_lin_random, fj_short_random, fs_random(2), fs_random_no_lp(2),
-    fs_random_quick_restart(2), fs_random_quick_restart_no_lp]
-  - 15 incomplete subsolvers: [feasibility_pump, graph_arc_lns, graph_cst_lns,
-    graph_dec_lns, graph_var_lns, packing_precedences_lns,
-    packing_rectangles_lns, packing_slice_lns, rins/rens, rnd_cst_lns,
-    rnd_var_lns, scheduling_precedences_lns, violation_ls(3)]
-  - 3 helper subsolvers: [neighborhood_helper, synchronization_agent,
-    update_gap_integral]
-- `solver.parameters.num_workers = 64`:
-  - 15 full problem subsolvers: [default_lp, lb_tree_search, max_lp, no_lp,
-    objective_lb_search, objective_lb_search_max_lp, objective_lb_search_no_lp,
-    objective_shaving_search_max_lp, objective_shaving_search_no_lp, probing,
-    probing_max_lp, pseudo_costs, quick_restart, quick_restart_no_lp,
-    reduced_costs]
-  - 47 first solution subsolvers: [fj_long_default(2), fj_long_lin_default(2),
-    fj_long_lin_perturb(2), fj_long_lin_random(2), fj_long_perturb(2),
-    fj_long_random(2), fj_short_default(2), fj_short_lin_default(2),
-    fj_short_lin_perturb(2), fj_short_lin_random(2), fj_short_perturb(2),
-    fj_short_random(2), fs_random(6), fs_random_no_lp(6),
-    fs_random_quick_restart(6), fs_random_quick_restart_no_lp(5)]
-  - 19 incomplete subsolvers: [feasibility_pump, graph_arc_lns, graph_cst_lns,
-    graph_dec_lns, graph_var_lns, packing_precedences_lns,
-    packing_rectangles_lns, packing_slice_lns, rins/rens, rnd_cst_lns,
-    rnd_var_lns, scheduling_precedences_lns, violation_ls(7)]
-  - 3 helper subsolvers: [neighborhood_helper, synchronization_agent,
-    update_gap_integral]
+| # Workers | Full Problem Subsolvers                                                        | First Solution Subsolvers                                                                                                                                                                                                                                                                 | Incomplete Subsolvers                                                                                                                                                                                                                                                  | Helper Subsolvers                                                                 |
```

Comment:
<img width="715" alt="image" src="https://github.com/user-attachments/assets/ff084c0c-7512-4c64-9b3d-4e228b9e9bca">

The table looks like this. I hope my assumption that the set of solvers are always contained in subsequent number of workers :-) I used GPT to help me with formatting so please double check.

Reply:
That is pretty!